scripts/typescript_checks.py

```diff
@@ -45,19 +45,26 @@
     'core/templates/App.ts',
     'core/templates/AppSpec.ts',
     'core/templates/Polyfills.ts',
+    'core/templates/app-events/event-bus.service.spec.ts',
```

Comment:
This undoes Jeevesh's GSoC project, it's not good to add a bunch of stuff to this file. Why are new issues with typing being introduced in this PR?

Reply:
These files are not tested by strict checks in develop. With my changes they were included in the checks and unfortunately I had to add a small portion of them to this list. This was discussed with Vojta.

Reply:
There are a lot of files added and I think our discussion last time about why they needed to be added was inconclusive. Could you please explain the root of the issue more? I think I would like @JeeveshGarg to sign off on this since this relates to his project.

/cc @JeeveshGarg @vojtechjelinek

Reply:
As per offline discussion with @srijanreddy98, we will make followup PR for removal of these files after merge of this PR, as due to some changes in global type, specific same error is occur is most of the files then in there dependencies files, so we need to add them here.

CC: @seanlip: As this PR is under high-priority, so giving permission to add.